### PR TITLE
fix: adjusted styling for medium screen size

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* adjusted styling for medium screen size
* modified breaking point for sidebar
* modified header styling
* colors in light theme

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes

**This pull request makes the following changes:**

**Fixes**

=Before=
image

image

image

=After=
768px
image

image

Figma
image

### CODE REVIEW

**Review by Kahonnohak — APPROVED** [review]
Thanks you!! You are a super star! 

Could you possibly quickly change the sidebar navi menu, Stakig → dApp Staking please? or should it be a different PR?

File: src/components/header/Header.vue [hunk 1]
```diff
@@ -143,6 +143,12 @@ export default defineComponent({
 
 .m-header {
   background: #fff !important;
+  height: 64px !important;
```
Comment (sirius651) [inline src/components/header/Header.vue:146]: could we adjust this styling on the ui library instead of overwritten css?

**Review by sirius651 — CHANGES_REQUESTED** [review]

File: src/components/header/Header.vue (hunk 1 above)
Comment (impelcrypto) [reply src/components/header/Header.vue:146]: Ok. How about fixing it in another PR?

Comment (sirius651) [reply src/components/header/Header.vue:146]: Sure

**Review by sirius651 — APPROVED** [review]

**Pull request merged**